test/java/seedu/addressbook/commands/DeleteCommandTest.java

```diff
@@ -145,18 +165,22 @@ private void assertDeletionFailsDueToNoSuchPerson(int visibleIndex, AddressBook
      *
      * @throws PersonNotFoundException if the selected person is not in the address book
      */
-    private void assertDeletionSuccessful(int targetVisibleIndex, AddressBook addressBook,
-                                          List<ReadOnlyPerson> displayList) throws PersonNotFoundException {
-
-        ReadOnlyPerson targetPerson = displayList.get(targetVisibleIndex - TextUi.DISPLAYED_INDEX_OFFSET);
-
+    private void assertMultipleDeletionSuccessful(AddressBook addressBook, List<ReadOnlyPerson> displayList, 
```

Comment:
What happens if one of the 2 ppl i want to delete is not found?

Reply:
Do you mean not found in the AddressBook? There are test cases for them. The test case is called 'assertDeletionFailsDueToNoSuchPerson'.
For every person that is not found, it will print out a line stating that the person is not found in the AddressBook.